en/qgc/video_streaming_wifi_broadcast.md

```diff
@@ -1,99 +1,118 @@
 # Long-distance video streaming in QGroundControl
 
-This page shows how to set up a a companion computer (Odroid C1 or C0) with a camera (Logitech C920) such that the video stream is transferred via the Odroid C1 to a network computer and displayed in the application QGroundControl that runs on this computer. This setup uses WiFi in unconnected (broadcast) mode.
+This page shows how to set up a a companion computer with a camera (Logitech C920 or RaspberryPI camera) such that the video stream is transferred from UAV to a ground computer and displayed in the application QGroundControl that runs on this computer. This setup uses WiFi in unconnected (broadcast) mode.
 
 The whole hardware setup consists of the following parts:
 
-On TX (copter) side:
-* Odroid C1
-* Logitech camera C920
+On TX (UAV) side:
+* NanoPI NEO2 (and/or Raspberry PI if use PI camera)
```

Comment:
"Raspberry Pi" (Pi is lower case everywhere). Links to manufacturer for components always useful for readers.